Makefile

```diff
@@ -1,5 +1,7 @@
 artifact_name       := test-data-generator
-version             := "unversioned"
+version             := unversioned
+docker_repository   := ${artifact_name}
```

Comment:
If this is now just a proxy for `${artifact_name}` I'd get rid of it and use `${artifact_name}` directly in the `docker-build` and `docker-run` targets

Reply:
Yes, fair point. Now removed the "docker_repository" variable.